scala/README.md

```diff
@@ -406,20 +485,12 @@ It is definitely recommended to read the full Twitter's "[Effective Scala]" guid
     val good = geo(latitude = a, longitude = b)
     ```
 
-0. Whenever possible, simplify pattern matching expressions by omitting the `match` keyword and using partial functions:
+0. Always pass named `Boolean` parameters to functions, even when they are the only parameter:
```

Comment:
meh i guess this is basically the point below. but maybe we can make it clear that this is a tip for when you are using another library that exposes a magic boolean?